src/scripts/elasticsearch-ubuntu-install.sh

```diff
@@ -116,7 +116,7 @@ STORAGE_KEY=""
 UBUNTU_VERSION=$(lsb_release -sr)
 
 #Loop through options passed
-while getopts :n:m:v:A:R:K:S:Z:p:a:k:L:C:B:Xxyzldjh optname; do
+while getopts :n:m:v:A:R:K:S:Z:p:a:k:L:C:B:E:Xxyzldjh optname; do
```

Comment:
Want to keep the provisioning of a storage account for azure-repository plugin separate.